paddle/fluid/framework/ir/mkldnn/cpu_quantize_squash_pass_tester.cc

```diff
@@ -119,56 +182,175 @@ void MainTest(const ProgramDesc& prog, int removed_nodes_num) {
   }
 
   graph->SetNotOwned(kParamScopeAttr, &scope);
-
   auto pass = PassRegistry::Instance().Get("cpu_quantize_squash_pass");
 
   int original_nodes_num = graph->Nodes().size();
-
   graph.reset(pass->Apply(graph.release()));
-
   int current_nodes_num = graph->Nodes().size();
 
   EXPECT_EQ(original_nodes_num - removed_nodes_num, current_nodes_num);
 }
 
+// check op->scale_out
+void EqualScaleOutTest(const ProgramDesc& prog, const std::string& name,
+                       float scale) {
+  std::unique_ptr<ir::Graph> graph(new ir::Graph(prog));
+
+  // Init scope, as it is used in pass
+  auto place = paddle::platform::CPUPlace();
+  NaiveExecutor exe{place};
+  Scope scope;
+  exe.CreateVariables(prog, 0, true, &scope);
+
+  for (auto& v : variable_names) {
+    InitTensorHolder(&scope, place, v.c_str());
+  }
+
+  graph->SetNotOwned(kParamScopeAttr, &scope);
+  auto pass = PassRegistry::Instance().Get("cpu_quantize_squash_pass");
+  graph.reset(pass->Apply(graph.release()));
```

Comment:
The code inside `CheckRequantScalesTest` looks the same as the one in lines above. Perhaps the part:
```C++
  std::unique_ptr<ir::Graph> graph(new ir::Graph(prog));

  // Init scope, as it is used in pass
  auto place = paddle::platform::CPUPlace();
  NaiveExecutor exe{place};
  Scope scope;
  exe.CreateVariables(prog, 0, true, &scope);

  for (auto& v : variable_names) {
    InitTensorHolder(&scope, place, v.c_str());
  }

  graph->SetNotOwned(kParamScopeAttr, &scope);
  auto pass = PassRegistry::Instance().Get("cpu_quantize_squash_pass");
  graph.reset(pass->Apply(graph.release()));
```
Could be extracted into function PrepareGraph which returns a Graph? It would also accept the pass name as a parameter (in this case `"cpu_quantize_squash_pass"`).

Reply:
Alternatively, if the NaiveExecutor and or Scope need to remain in the same scope as graph does, perhaps it'd also be an option to pass the code that follows as a lambda:
```C++
[]() { for (auto* node : graph->Nodes()) {
    if (node->IsOp() &&
  ....
}
```